src/utils.c

```diff
@@ -1788,6 +1788,9 @@ void utils_tidy_path(gchar *filename)
 	const gchar *needle;
 	gboolean preserve_double_backslash = FALSE;
 
+	if (utils_is_uri(filename))
+		return;
```

Comment:
@techee did you notice the release date THIS MONTH, of course nothing omits it at the moment, its only just been allowed.